decidim-meetings/app/controllers/decidim/meetings/polls/answers_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
           enforce_permission_to(:create, :answer, question:)
           @form = form(AnswerForm).from_params(params, question:, current_user:)
```

Comment:
This needs to be changed so the `current_user` is available in the form instance.

```suggestion
          @form = form(AnswerForm).from_params(params.merge(question:, current_user:))
```